docs/_includes/js/modal.js

```diff
@@ -0,0 +1,41 @@
+(function() {
+
+	var showModal = document.querySelectorAll('.showModal'),
+      youtube   = document.getElementsByTagName("iframe")[0];
+
+	var initialize = function() {
+    [].forEach.call(showModal, function(el) {
+      el.addEventListener('click', function(e) {
+        var close, overlay, modal;
+
+        overlay = document.querySelector('.Overlay');
+        modal   = document.querySelector('.Modal');
+        close   = document.querySelector('.u-modalClose');
+
+        overlay.classList.add('isShown');
+        modal.classList.add('isShown');
+
+        if (youtube) {
+          toggleVideo(youtube, 'show');
+        }
+
+        [].forEach.call([close, overlay], function(el) {
+          el.addEventListener('click', function(e) {
+            overlay.classList.remove('isShown');
+            modal.classList.remove('isShown');
+            if (youtube) {
+              toggleVideo(youtube, 'hide');
+            }
+          })
+        });
+      });
+    })
+	}
```

Comment:
The modal's being used on the Trials page, and will be used on the docs training video page too. Where would be the best place to put this as a util? And what should it be called then?